apis/opts/memory_swappiness.go

```diff
@@ -6,7 +6,7 @@ import "fmt"
 
 // ValidateMemorySwappiness verifies the correctness of memory-swappiness.
 func ValidateMemorySwappiness(memorySwappiness int64) error {
-	if memorySwappiness < 0 || memorySwappiness > 100 {
+	if memorySwappiness != -1 && (memorySwappiness < 0 || memorySwappiness > 100) {
```

Comment:
how about `if  (memorySwappiness < -1 || memorySwappiness > 100)`

Reply:
Do not do this, [0, 100] is a valid value, -1 is the value we enable to follow with moby.

Reply:
OK, You are the boss :)